util/settings/settings.go

```diff
@@ -112,6 +114,7 @@ const (
 	settingDexConfigKey = "dex.config"
 	// settingsOIDCConfigKey designates the key for OIDC config
 	settingsOIDCConfigKey = "oidc.config"
+	statusBadgeEnabledKey = "statusbadge.enabled"
```

Comment:
Probably should add a comment similar to all the other constants here.

Reply:
done